index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8"/>
+        <title>Weather</title>
+        <link rel="stylesheet" href="styles/index.css"/>
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.11.2/css/all.css" 
+        integrity="sha256-46qynGAkLSFpVbEBog43gvNhfrOj+BmwXdxFgVK/Kvc=" crossorigin="anonymous" />  
+        <link rel="preconnect" href="https://fonts.gstatic.com">
+        <link href="https://fonts.googleapis.com/css2?family=Caveat:wght@657&display=swap" rel="stylesheet"> 
+    </head>
+    <body>
+        <header>
+            <section class = "city-header">
+                <h1><em class= "your-city">your</em> Weather Report</h1>
+            </section>
+        </header>
+        <main>
+            <section class = "weather-forms">
+                <div class="weather-boxes">
+                    <h3>City</h3>
+                        <input type="text" id="city-input">
+                        <button id="weather-button">reset</button>
+                </div>
+
+                <div class= "weather-boxes">
+                    <h3>Sky</h3>
+                        <select>
+                            <option>Sunny</option>
+                            <option>Cloudy</option>
+                            <option>Rainy</option>
+                            <option>Snowy</option>
+                        </select>
+                </div>
+
+                <div class= "weather-boxes">
+                    <h3>Temperature</h3>
+                    <!--clickable option to increase or decrease temp hot cold--->
+                        <section id="tempbox">
+                            <p id ="decrease-temp">🔽</p>
+                            <p id=temp>70</p>
+                            <p id="increase-temp">🔼</p>
```

Comment:
I just want to point out that making these `p` tags is not great for accessibility.  Instead I suggest making them hyperlinks  or buttons.